test/e2e_node/disk_eviction_test.go

```diff
@@ -142,18 +136,30 @@ var _ = framework.KubeDescribe("Kubelet Eviction Manager [Flaky] [Serial] [Disru
 						recordContainerId(containersToCleanUp, podData.Status.ContainerStatuses)
 
 						if podData.Status.Phase != api.PodRunning {
-							return fmt.Errorf("expected phase to be running. got %+v", podData.Status.Phase)
+							err = verifyPodEviction(podData)
+							if err != nil {
+								return err
+							}
 						}
-
 						evictionOccurred = true
+						return fmt.Errorf("waiting for node disk pressure condition to be set")
+					}
+
+					// The node should have disk pressure condition after the pods are evicted.
+					if !nodeDiskPressureCondition {
+						if !nodeHasDiskPressure(f.Client) {
```

Comment:
The rearrangement is to make the code cleaner (I hope)

The node's disk pressure can only be relieved after 1 minute the disk pressure has been set per the configuration.